src/defaultsettings.cpp

```diff
@@ -57,6 +57,10 @@ void set_default_settings(Settings *settings)
 	settings->setDefault("curl_verify_cert", "true");
 	settings->setDefault("enable_remote_media_server", "true");
 	settings->setDefault("enable_client_modding", "false");
+	settings->setDefault("enable_knockback", "true");
+	settings->setDefault("knockback_default_duration", "0.3");
+	settings->setDefault("knockback_horizontal", "17.0");
+	settings->setDefault("knockback_vertical", "7.5");
```

Comment:
Defined horizontal and vertical components will look wrong (and it does currently in the video), it depends on attack direction.